src/Directory.Build.props

```diff
@@ -22,8 +22,8 @@
 
   <PropertyGroup>
     <!-- These are the versions of the things we are CREATING in this repository -->
-    <PerfViewVersion>2.0.48</PerfViewVersion>
-    <TraceEventVersion>2.0.48</TraceEventVersion>
+    <PerfViewVersion>2.0.49</PerfViewVersion>
+    <TraceEventVersion>2.0.49</TraceEventVersion>
```

Comment:
Let's make this a separate change, since it's not logically part of this fix.

Reply:
Sure.